_layouts/default.html

```diff
@@ -124,10 +131,15 @@ <h2 id="title">
       </div>
       <script type="text/javascript" src="{{ "/assets/js/darkmode.js" | relative_url }}"></script>
```

Comment:
I believe this script would currently get executed twice.

Reply:
Thanks, should be fixed now.